examples/sensor_metadata_schema/sensor_metadata.json

```diff
@@ -0,0 +1,32 @@
+[{
```

Comment:
I would not put all in examples, but rather in digitaltwin/metadata_schema/sensor.json. This is then included in the general pip package, the example folder is only for showing how the "real" library is used. The schema itself is then part of the package, the usage of a specific example is then in examples.